src/main/java/org/fogbowcloud/sebal/SEBALHelper.java

```diff
@@ -226,369 +209,35 @@ protected static UTMCoordinate convertLatLonToUtm(double latitude, double longit
-
-		gdal.AllRegister();
-
-		Dataset dst = gdal.Open(fmaskFilePath, gdalconstConstants.GA_ReadOnly);
-		org.gdal.gdal.Band band = dst.GetRasterBand(1);
-
-		double[] fmask = new double[maskWidth * maskHeight];
-		band.ReadRaster(iInitial, jInitial, maskWidth, maskHeight, fmask);
-
-		return fmask;
-	}
-	
-	public static DefaultImagePixel readElevation(ImagePixel imagePixel) {
-
-		DefaultImagePixel defaultImagePixel = new DefaultImagePixel();
-
-		defaultImagePixel.z(imagePixel.z());
-		GeoLoc geoLoc = new GeoLoc();
-
-		geoLoc.setLat(imagePixel.geoLoc().getLat());
-		geoLoc.setLon(imagePixel.geoLoc().getLon());
-
-		defaultImagePixel.geoLoc(geoLoc);
-
-		return defaultImagePixel;
-	}
-
-	public static long getDaysSince1970(String mtlFilePath) throws Exception,
-			ParseException {
-		Product product = readProduct(mtlFilePath, null);
-		MetadataElement metadataRoot = product.getMetadataRoot();
-		String dateAcquiredStr = metadataRoot.getElement("L1_METADATA_FILE")
-				.getElement("PRODUCT_METADATA").getAttribute("DATE_ACQUIRED").getData()
-				.getElemString();
-	
-		DateFormat format = new SimpleDateFormat("yyyy-MM-dd");
-		Date date1970 = format.parse("1970-01-01");
-		Date dateAcquired = format.parse(dateAcquiredStr);
-	
-		long daysSince1970 = (dateAcquired.getTime() - date1970.getTime()) / (24 * 60 * 60 * 1000);
-		return daysSince1970;
+		return new UTMCoordinate(easting, northing);
 	}
 
-	public static String getAllPixelsFilePath(String outputDir, String mtlName, int iBegin, int iFinal,
-			int jBegin, int jFinal) {
-		if (mtlName == null || mtlName.isEmpty()) {
-			return outputDir + "/" + iBegin + "." + iFinal + "." + jBegin + "."
-					+ jFinal + ".pixels.csv";
-		}
-		return outputDir + "/" + mtlName + "/" + iBegin + "." + iFinal + "." + jBegin + "."
-				+ jFinal + ".pixels.csv";
-	}
-	
-	public static String getWeatherFilePath(String outputDir, String mtlName, String imageFileName) {
+	public static String getWeatherFilePath(String outputDir, String mtlName,
+			String imageFileName) {
 		if (mtlName == null || mtlName.isEmpty()) {
 			return outputDir + File.separator + imageFileName + "_station.csv";
 		} else {
-			return outputDir + File.separator + mtlName + File.separator + imageFileName + "_station.csv";
+			return outputDir + File.separator + mtlName + File.separator + imageFileName
+					+ "_station.csv";
 		}
 	}
-	
-	public static String getElevationFilePath(String outputDir, String mtlName, String imageFileName,
-			int iBegin, int iFinal, int jBegin, int jFinal) {
-		if (mtlName == null || mtlName.isEmpty()) {
-			return outputDir + "/" + imageFileName + "_" + iBegin + "." + iFinal + "." + jBegin + "."
-					+ jFinal + ".elevation.tiff";
-		} else {
-			return outputDir + "/" + mtlName + "/" + imageFileName + "_" + iBegin + "." + iFinal
-					+ "." + jBegin + "." + jFinal + ".elevation.tiff";
-		}
-	}
-	
-	public static String getDadosFilePath(String rScriptFilePath) {
-		return rScriptFilePath  + "dados.csv";
-	}
 
```

Comment:
what is file does not exists or it is null? do we fail in silence? I think we should not